lib/puppetserver/ca/action/prune.rb

```diff
@@ -45,31 +54,57 @@ def run(inputs)
           puppet.load(logger: @logger)
           return 1 if Errors.handle_with_usage(@logger, puppet.errors)
 
+          # Validate arguments
+          if (remove_entries && (!serialnumbers && !certnames))
+            return 1 if Errors.handle_with_usage(@logger,["--remove-entries option require --serial or --certname values"])
+          end
+
+          # Validate inventory path
+          if remove_entries && certnames
+             errors = FileSystem.validate_file_paths(puppet.settings[:cert_inventory])
```

Comment:
I'm not sure that a missing inventory file should be a fatal condition?  Given that we can potentially find the names in the certs, we can still try.  Maybe this should be a warning?

Reply:
Oh, that's true.

Reply:
Updates code and tests